src/main/java/com/google/sps/servlets/AccountData.java

```diff
@@ -18,16 +18,30 @@
 /** Servlet that returns all account related info (links to login pages/account info). */
 @WebServlet("/account")
 public class AccountData extends HttpServlet {
+  private final String idProperty = "id";
```

Comment:
This can be static since it's a constant value. Let's also add constants for the other property names ("email", "first-name", "last-name") for consistency.

Reply:
I'm not sure but you might be confusing final with static? I'm not accessing the constant anywhere outside the class, so not sure why static would be necessary.

Reply:
It should be both static+final. The combination is often used for things that effectively constants, because 1) they're shared by all instances of the class (static) and 2) they cannot be changed (final).

With what you have now (`private final String idProperty = "id";`), each instance of this class would get it's own copy of the variable which isn't necessary. It's kind of a trivial difference in this case, because there would only be one instance of the class anyway, but it's good practice to use static and/or final wherever possible.

Also from the style guide, static+final variables should use screaming case so it should be:
`private static final String ID_PROPERTY = "id";`
(https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names)